test/soll-runtime-test/evmc.py

```diff
@@ -0,0 +1,183 @@
+import ctypes
```

Comment:
We should use pylint or other python linter for formatting this file.

Reply:
I have applied https://github.com/hhatto/autopep8 on all python files